spec/run_spec.js

```diff
@@ -0,0 +1,54 @@
+describe('run', function() {
+  var path = require('path'),
+      runJasmine = require('../lib/run');
+
+  it('should go through each jasmine method', function() {
+    var fakeJasmine = jasmine.createSpyObj('jasmine', ['loadConfigFile', 'addHelperFiles', 'addRequires', 'showColors', 'execute', 'stopSpecOnExpectationFailure',
+      'stopOnSpecFailure', 'randomizeTests', 'seed', 'coreVersion', 'clearReporters', 'addReporter']),
+        env = {
+          configPath: 'somewhere.json',
+          stopOnFailure: true,
+          failFast: true,
+          seed: 12345,
+          random: true,
+          helpers: 'helpers/**/*.js',
+          requires: 'requires/**/*.js',
+          reporter: path.resolve(path.join(__dirname, 'fixtures', 'customReporter.js')),
+          color: true,
+          files: 'specs/**/*.spec.js',
+          filter: 'filter'
+        };
+    runJasmine(fakeJasmine, env, console.log);
+
+    expect(fakeJasmine.loadConfigFile).toHaveBeenCalledWith(env.configPath);
+    expect(fakeJasmine.stopSpecOnExpectationFailure).toHaveBeenCalledWith(env.stopOnFailure);
+    expect(fakeJasmine.stopOnSpecFailure).toHaveBeenCalledWith(env.failFast);
+    expect(fakeJasmine.seed).toHaveBeenCalledWith(env.seed);
+    expect(fakeJasmine.randomizeTests).toHaveBeenCalledWith(env.random);
+    expect(fakeJasmine.addHelperFiles).toHaveBeenCalledWith(env.helpers);
+    expect(fakeJasmine.addRequires).toHaveBeenCalledWith(env.requires);
+    expect(fakeJasmine.clearReporters).toHaveBeenCalled();
+    expect(fakeJasmine.addReporter).toHaveBeenCalled();
+    expect(fakeJasmine.showColors).toHaveBeenCalledWith(env.color);
+    expect(fakeJasmine.execute).toHaveBeenCalledWith(env.files, env.filter);
+  });
+
+  it('should not go through each jasmine method', function() {
```

Comment:
From this description, I can't tell _why_ we're not going "through each jasmine method". Please expand.